api/src/controllers/users.ts

```diff
@@ -501,4 +501,42 @@ router.post(
 	respond,
 );
 
+const registerSchema = Joi.object<{ email: string; password: string }>({
+	email: Joi.string().email().required(),
+	password: Joi.string().required(),
+});
+
+router.post(
+	'/register',
```

Comment:
It'd be nice if we would add a way to specify more user fields because I can see people using this service to build their own login pages with more fields :thinking: 

Allowing other user fields would need to be sanitized though we cant let them override role for example

Reply:
You mean like require a phone number or similar? 🤔

Reply:
Yes for example a number or even simpler first & last name. Avatar could also be useful, stuff like that. It'd be nice to also allow populating user-provided fields for when users have manually extended the `directus_users` table so it'd have to be dynamic but I'm a little hesitant there because of security concerns. Mhh

Reply:
Interesting ❤️ Well, actually it would "just" be another (public/shared) form where we could use existing validation logic, etc. 🤔